<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>321</width>
    <height>145</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="navBar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>https://google.com</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openBrowser">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>openBrowser</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>navBar</string>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="architekLink">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>193</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>architek.co</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
